### PR TITLE
Travis improvements

### PR DESCRIPTION
**Pull request by thefosk**

### CODE REVIEW

File: spec/spec_helpers.lua [hunk 1]
```diff
@@ -60,10 +60,17 @@ local function kong_bin(signal, conf_file, skip_wait)
     error("spec_helper cannot "..signal.." kong: \n"..result)
   end
 
-  if signal == "start" and not skip_wait then
-    os.execute("while ! [ -f "..env.configuration.pid_file.." ]; do sleep 0; done")
-  elseif signal == "quit" or signal == "stop" then
-    os.execute("while [ -f "..env.configuration.pid_file.." ]; do sleep 0; done")
+<<<<<<< HEAD
```
Comment (thibaultCha) [inline spec/spec_helpers.lua]: haha

**Comment by thibaultCha** [comment]
Not having this (for start and stop as well) breaks the tests for me, hence why I added stop back.

**Comment by thefosk** [comment]
Removing it didn't break the tests for me locally, which makes me assume that `nginx start` returns before the server is able to process requests.

**Comment by thibaultCha** [comment]
Exactly, I really think we need this, even if I hate it. If we find a different way, it'll be most welcomed.

**Pull request closed without merging**